modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/helper/EndpointHelper.java

```diff
@@ -124,6 +125,34 @@ public Page<Map<String, Object>> getResponseEntityMapsPage(
 			responseEntityMaps, pagination, objectEntriesPage.getTotalCount());
 	}
 
+	public Map<String, Object> postObjectEntry(
+			long companyId, Map<String, Object> propertiesMap,
+			APIApplication.Schema requestSchema,
+			APIApplication.Schema responseSchema, String scopeKey)
+		throws Exception {
+
+		Map<String, Object> objectPropertiesMap = new HashMap<>();
```

Comment:
I think Brian does not like calling just `object`, that about `objectEntryProperties`?